src/test/java/PenguinTest.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * This file contains sample JUnit test cases for Horse.java
+ * You should NOT modify this file!
+ */
+
+import org.junit.*;
+
+import static org.junit.Assert.*;
+
+
+public class PenguinTest {
+    Penguin Penguin1;
```

Comment:
bad naming convention, should begin w lowercase

Reply:
Yes